h/search/__init__.py

```diff
@@ -29,8 +31,10 @@ def includeme(config):
     # Add a property to all requests for easy access to the elasticsearch
     # client. This can be used for direct or bulk access without having to
     # reread the settings.
-    config.registry['es.client'] = get_client(settings)
+    config.registry['es.client'] = get_client_old(settings)
     config.add_request_method(
         lambda r: r.registry['es.client'],
         name='es',
         reify=True)
+    # Set the elasticsearch client connection as the default connection.
+    connections.add_connection('default', get_client(settings))
```

Comment:
We could do this inside of the get_client but I think it's nice to still have a get_client function for testing purposes. The using param is set to 'default' by default in elasticsearch-dsl so this is another way to set the default without creating a client via elasticsearch-dsl. Both ways are perfectly valid.